tests/database/test_provider_collector.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Test the ProviderDBAccessor utility object."""
+
+from masu.database.provider_collector import ProviderCollector
+from tests import MasuTestCase
+
+
+class ProviderQueryTest(MasuTestCase):
+    """Test Cases for the ProviderDBAccessor object."""
+
+    def setUp(self):
+        pass
+
+    def test_initializer(self):
+        '''Test Initializer'''
```

Comment:
Nit, but can we be consistent with quotes?
``'''Test Initializer''' `` --> ``"""Test Initializer"""``